packages/client/src/views/SysAdmin/Config/Certificates/previewDummyData.ts

```diff
@@ -17,7 +17,7 @@ const dummyBirthRegistrationResponse = {
   _fhirIDMap: {
     composition: '1e09f4b5-43f4-415c-b2ab-f24a4b0be20a',
     encounter: '65f436b8-562a-4a35-8df9-ff41205a7770',
-    eventLocation: '5c6abc88-26b8-4834-a1a6-2992807e3a72',
+    eventLocation: 'ec396045-3437-4224-8e03-f299e17158e5',
```

Comment:
@sadmananik unfortunately this is not sustainable.  Whenever a country updates their Health Facilities or location lists, these IDs will change. Instead write a function that gets any health location ID from offline data

Reply:
Actually, this eventLocation ID is coming from mock data so it will be fixed and work always even if DB chnage.

Reply:
@euanmillar  I think @sadmananik used the ids defined herein `mockOfflineData`, so backup updates wont effect this 
 https://github.com/opencrvs/opencrvs-core/blob/300ff299c658b6e3eb2e4acc96de5db72e259289/packages/client/src/views/SysAdmin/Config/Certificates/previewDummyData.ts#L1221

Reply:
Sorry but it has to be a function even on the mocked data

Reply:
@sadmananik @tahmidrahman-dsi I still think that this is not going to work.  Because you have a health facility, and it must appear on the certificate next to the district and state.  But the partfOf ids for location and state will use offline data to render the state and district name.  These IDs will change.   I think that the mock certificate data location ids must be generated from functions.  I will work on a fix.